staging/src/kubevirt.io/client-go/kubecli/kubevirt.go

```diff
@@ -84,6 +85,7 @@ type KubevirtClient interface {
 	GuestfsVersion() *GuestfsVersion
 	RestClient() *rest.RESTClient
 	GeneratedKubeVirtClient() generatedclient.Interface
+	CoreClient() *kvcorev1client.Clientset
```

Comment:
Why do we need this?

Reply:
Leftover :man_facepalming: Removed entirely the commit